src/components/TreeGrid.ts

```diff
@@ -145,11 +147,27 @@ const TreeGridInner = <T extends RowContents>(props: TreeGridPropsInner<T>) => {
     setData(_.set(`[${getRowIndex(row, data)}].state`, 'closed', data));
   };
 
+  /*
+     Height is the window height minus:
+     64 px for the top bar
+     120 px for the dataset builder header
+     73 px for the domain header
+     54 px for the column headers
+     We want to have the height clamped here, in order to allow the grid
+     to handle the scrolling, so that we can have initial scrolling behavior
+     on the grid
+     */
+  const gridHeight = window.innerHeight - 64 - 120 - 73 - 54;
```

Comment:
this seems brittle as if the height of any of these components change, this will no longer be correct. Is there a way to derive the heights of these components from them instead of hardcoding what they currently are?

Reply:
Let me poke around, there might be something I can do.